RefactoringPracticeI/GameInventory/Inventory/Handlers/GenericItemHandler.cs

```diff
@@ -0,0 +1,20 @@
+namespace RefactoringExerciseI.Inventory.Handlers
+{
+    class GenericItemHandler : ItemUpdateHandler
+    {
+        public override void UpdateItem(Item item)
```

Comment:
is this not going to call the `next?.UpdateItem(item);`? or this is the last link on the chain

Reply:
GenericItemHandler should always be the last handler in the chain